freqtrade/exchange/coinmate.py

```diff
@@ -0,0 +1,121 @@
+"""Coinmate exchange subclass"""
+
+import logging
+from typing import Dict
+
+from freqtrade.enums import MarginMode, TradingMode
+from freqtrade.exceptions import OperationalException
+from freqtrade.exchange import Exchange
+from freqtrade.exchange.exchange_types import FtHas
+
+logger = logging.getLogger(__name__)
+
+
+class Coinmate(Exchange):
+    """
+    Coinmate exchange class. Contains adjustments needed for Freqtrade to work
+    with this exchange.
+    
+    Coinmate is a Czech cryptocurrency exchange supporting spot trading only.
+    Rate limit: 100 requests/minute.
+    """
+
+    _ft_has: FtHas = {
+        # Core trading features
+        "stoploss_on_exchange": False,  # No stop-loss support on exchange
+        "order_time_in_force": ["GTC"],  # Good Till Cancelled only
+        "trades_pagination": "id",
+        "trades_pagination_arg": "fromId",
+        "trades_has_history": True,
+        
+        # Order book features  
+        "l2_limit_range": [1, 10, 50, 100],  # Supported order book depths
+        "l2_limit_range_required": False,
+        
+        # Market data features
+        "ohlcv_candle_limit": None,  # No OHLCV support
+        "ohlcv_has_history": False,
+        
+        # Rate limiting
+        "ccxt_futures_name": "swap",  # Required by base class
+    }
+
+    _supported_trading_mode_margin_pairs: list[tuple[TradingMode, MarginMode]] = [
+        # Coinmate only supports spot trading
+        (TradingMode.SPOT, MarginMode.NONE),
+    ]
+
+    @property
+    def _ccxt_config(self) -> Dict:
+        """Configure CCXT parameters for Coinmate"""
+        config = super()._ccxt_config
+        config.update({
+            "uid": self._api_key_headers.get("uid", ""),
+            "rateLimit": 600,  # 100 requests/minute
```

Comment:
we don't force options like this (applies to ALL settings here).
For the rate-limit specifically - if they're really wrong (test without any custom rate-limit settings) - report it to ccxt.

If this is required _AND_ ccxt refuses to fix this, add it to the documentation
All other options - remove.
we won't force time-adjustments - and uid is already read from the configuration as part of the base exchange.

(in summary - this whole property is useless and has to be removed.